pandas/core/window/exponentialmoving.py

```diff
@@ -185,7 +185,7 @@ def __init__(
 
     @property
     def _constructor(self):
-        return EWM
+        return EWF
```

Comment:
this still need to change too?

Reply:
Tell you what mate give me some time I'll rollback all the changes, start afresh and create a new pull request. i've confused myself. thanks for your help!